external/wakaama/core/er-coap-13/er-coap-13.h

```diff
@@ -92,7 +92,11 @@
 
 /* Bitmap for set options */
 enum { OPTION_MAP_SIZE = sizeof(uint8_t) * 8 };
-#define SET_OPTION(packet, opt) ((packet)->options[opt / OPTION_MAP_SIZE] |= 1 << (opt % OPTION_MAP_SIZE))
+#define SET_OPTION(packet, opt)	\
+	if (opt / OPTION_MAP_SIZE > COAP_OPTION_PROXY_URI / OPTION_MAP_SIZE) {	\
```

Comment:
Isn't the expression above using same denominator (OPTION_MAP_SIZE)? Even if it is required, since it is value 8, you could bit shift it by 3, which works out faster.